GirafIntegrationTest/tests/test_activity_controller.py

```diff
@@ -80,12 +84,16 @@ def test_activity_set_new_user_activity(self):
 
         Endpoint: POST:/v2/Activity/{user_id}/{weekplan_name}/{week_year}/{week_number}/{week_day_number}
         """
+        self.skipTest("Skipping since endpoint is broken")
```

Comment:
Use the decorator from `unittest` for this instead

```py
from unittest import skip

@skip('Skipping since endpoint is broken')
def test_method():
    # code
```